src/patreon.js

```diff
@@ -1,7 +1,7 @@
 import fetch from 'isomorphic-fetch'
 import { normalizeRequest, checkStatus, getJson } from './utils'
 
-function patreon(accessToken, config) {
```

Comment:
Just a note: plan was to allow passing in custom hostname and all that to support requesting against a mock/non-prod environment.

Reply:
sounds good, but i'd like to leave that out until it's actually used